hackIT/static/styles.css

```diff
@@ -11,54 +11,25 @@ nav{
 	box-shadow: 0px 1px 1px black;
 }
 
-span {
-	color:#e0e0e0;
-	text-decoration: underline;}
-
-#navLogo{
-	font-size: 32px;
-	color: #09cdda;
-	padding-left: 10px;
-
-	font-weight: 200;
-	font-family: 'Roboto Condensed', sans-serif;
-}
+.wrapper{
+	font-family: Share Tech Mono;
+	margin-right: auto;
+	margin-left: auto;
 
-#welcomeDiv{
-	background-color: #24464D;
-	border-radius: 10px;
-	box-shadow: 3px 3px 4px black;
-	width: 600px;
-	height: 400px;
-	border: 2px solid #02676e;
+	max-width: 900px;
 
-	margin-left:100px;
-	margin-top: 100px;
-
-	padding:10px;
+	padding-right: 10px;
+	padding-left: 10px;
+	background-color: #09CDDA;
```

Comment:
I think having the text in #09CDDA and the background the same as body would be more subtle for the eyes. Then we get a more "dark mode" experience

Having this #09CDDA for the backround of `Welcome to hackIT` or just the `hackIT` part would be cool tho. And would focus attention to the logo which might be good (honestly no clue).

Reply:
i changed the background and tried what you suggested about the header, i think it looks kind of nice so decided to keep it. The index site as a whole might need a bit more work though but it looks better now!